packages/gestalt/src/VideoPlayhead.js

```diff
@@ -38,6 +38,7 @@ export default class VideoPlayhead extends PureComponent<Props, State> {
     }
   };
 
+  // eslint-disable-next-line class-methods-use-this
```

Comment:
I agree with this rule in general, but don't think it's worth worrying about here. As with `react/sort-comp`, this will fall away as we refactor our few remaining class components.